contacts/models.py

```diff
@@ -39,3 +39,6 @@ class Address(models.Model):
 
     def __str__(self):
         return self.country
+
+    def get_absolute_url(self):
```

Comment:
why address has the same url as contact?

Reply:
Because after adding a contact or an address against the contact, we want to land it on ```/contacts```

Reply:
`get_absolute_url()` would only make sense when you want to land on the same model's list page. better to use `success_url` otherwise.